src/components/AllCreaturesTile.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+const AllCreaturesTile = props => {
+  return (
+    <div>
+      <Link to={`adoptable/${props.type}`}>
+      <h3>{props.name}</h3>
```

Comment:
The h3 and img tags are are children of the Link tag so they need to be indented.

Reply:
This still needs to be changed.

Reply:
This still needs to be changed. I cannot merge until this is fixed.

Reply:
we have the changes pushed ahead on my machine, but it isn't showing here.